scania-digital-design-system/tegel
# build(node): version 20

Setting node version 20 as min version. This is needed for us to be able to bump the stencil versions. 


**Solving issue**  
Fixes: -

**How to test**  
1. Check out branch
2. Try running install with node 18 in root - it should fail
3. Try running install with node 20 in root - it should succeed.
